mackerel-plugin-squid/lib/squid.go

```diff
@@ -42,7 +79,12 @@ func (m SquidPlugin) FetchMetrics() (map[string]interface{}, error) {
 		return nil, err
 	}
 	fmt.Fprintln(conn, "GET cache_object://"+m.Target+"/info HTTP/1.0\n\n")
-	scanner := bufio.NewScanner(conn)
+	return m.ParseMgrInfo(conn)
+}
+
+// ParseMgrInfo parser for squid mgr:info
```

Comment:
Can you add links to reference of `/info` ?
(I guess https://wiki.squid-cache.org/Features/CacheManager/Info?highlight=%28Feature..Squid.Cache.Manager%29)

Reply:
add url link at b1141bba7a99c1c8b68166f291f920556d8bcf40